### ISSUE TITLE
Math Field input is not accessible

### ISSUE THREAD

**Issue by savvaskevinreilly** [arnog/mathlive#1784]
## Description
When the math field is focused, screen readers do not announce the contents and instead announce "blank". Additionally, the MathLive keyboard shortcuts for reading the contents of the field do not work while a screen reader is running.

## Steps to Reproduce
1. Open NVDA (Windows), ChromeVox (ChromeOs) or VoiceOver (MacOS).
2. Open the demo page (https://cortexjs.io/mathlive/demo/) in Chrome browser.
3. Use keyboard navigation to set focus on the math field.
4. Observe the screen reader announces "blank" for the field's contents.
5. Use the keyboard shortcut, crtl + alt + up, to have MathLive read the contents.
6. Observe nothing it read aloud.
7. Close NVDA (Windows), ChromeVox (ChromeOs) or VoiceOver (MacOS).
8. Use the keyboard shortcut, crtl + alt + up, to have MathLive read the contents.
9. Observe the math field's content are now read aloud.

### Actual Behavior
The contents of the MathLive math field cannot be read aloud with the MathLive keyboard shortcuts unless any running screen reader software is first closed.

### Expected Behavior
The full contents of the MathLive math field should be readable by a screen reader upon focus. At a minimum, the MathLive keyboard shortcuts should work while a screen reader is running. It is not realistic to expect a screen reader user to close their screen reading software in order to know what the contents of the math field is.

**Comment by arnog** [comment]
I tried to follow the steps above using VoiceOver. At step (6), the content of the mathfield was read aloud, as expected.

Note that screen readers may in some cases intercept "ctrl+alt" keyboard shortcuts for their own purpose. In this case, the MathLive keyboard shortcut "ctrl+alt+up" may not work.

If using VoiceOver, go to Settings > VoiceOver > Open VoicOver Utility > General and in "Keys to use as the VoiceOver modifier" select "Capslock".

Alternatively, you can define a new keyboard shortcut for the mathfield that would not conflict with ones used by the screen reader or other system software.
